packages/neovim/src/testUtil.ts

```diff
@@ -19,8 +18,10 @@ export function findNvimOrFail() {
 
 const nvimPath = findNvimOrFail();
 
-let proc: cp.ChildProcessWithoutNullStreams;
-let nvim: NeovimClient;
+// eslint-disable-next-line import/no-mutable-exports
+export let proc: cp.ChildProcessWithoutNullStreams;
+// eslint-disable-next-line import/no-mutable-exports
+export let nvim: NeovimClient;
```

Comment:
can/should this be a function? it's a bit "magical" as a global var, and also makes debugging a bit more painful.

Reply:
I'll handle this too in a follow up PR :)